portal/v2/src/ClusterDetailListComponents/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,229 @@
+import { useState } from "react"
+import { ICluster } from "../../App"
+import { Modal, Stack, Text, IStackStyles, IconButton, mergeStyleSets, getTheme, Label, ThemeProvider, IStackTokens, PartialTheme, DefaultPalette} from "@fluentui/react"
+import { useBoolean } from "@fluentui/react-hooks"
+import { StatisticsWrapper } from "./StatisticsWrapper";
+
+import { GraphOptionsComponent} from "./GraphOptionsComponent"
+
+export const iconButtonStyles = mergeStyleSets({
+  icon: {
+    color: "white"
+  },
+  root: {
+    selectors: {
+      ':hover .ms-Button-icon': {
+        color: DefaultPalette.accent
+        
+      },
+    },
+  },
+})
+
+const global = new Date()
+export function Statistics(props: {
+    currentCluster: ICluster
+    detailPanelSelected: string
+    loaded: boolean,
+    statisticsType: string
+  }) {
+    const [globalDuration, setGlobalDuration] = useState<string>("1h")
+    const [globalEndDate, setGlobalEndDate] = useState<Date>(global)
+
+    function GlobalGraphOptionsBar() {
+      const stackStyles: IStackStyles = {
+        root: [
+          {
+            width: "100%",
+            padding: 0,
+          },
+        ],
+      }
+      const stackNavStyles: IStackStyles = {
+        root: {
+          padding: "0px 15px",
+          height: 40,
+        },
+      }
+      const containerStackTokens: IStackTokens = {}
+      const appStackTokens: IStackTokens = { childrenGap: 10 }
+      return(
+        <>
+          <Stack styles={stackStyles} tokens={containerStackTokens} horizontalAlign={"stretch"}>
+            <ThemeProvider theme={darkTheme}>
+              <Stack
+                grow
+                tokens={appStackTokens}
+                horizontalAlign={"start"}
+                verticalAlign={"center"}
+                horizontal
+                styles={stackNavStyles}>
+                <Stack.Item grow>
+                  <Text>{"Global Graph Options"}</Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <GraphOptionsComponent duration={globalDuration} setDuration={setGlobalDuration} endDate={globalEndDate} setEndDate={setGlobalEndDate}/>
+                </Stack.Item>
+              </Stack>
+            </ThemeProvider>
+          </Stack>
+        </>
```

Comment:
The fragment `<> </>` around this is not needed as it has a single root element.

Reply:
done